for-research/public-likelihoods/public-likelihoods.ipynb

```diff
@@ -0,0 +1,936 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3246ef2d-4f29-4082-9d4c-7fe3c0abf61e",
+   "metadata": {},
+   "source": [
+    "# Analysing statistical workspaces with xRooFit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "25deda8b-dc32-41f7-80be-4badd58f52a9",
+   "metadata": {},
+   "source": [
+    "# 1. Exploring workspaces\n",
+    "\n",
+    "This notebook covers how to open a workspace and explore its content using the xRooFit API. All workspace exploration involve using the [xRooNode](https://root.cern.ch/doc/master/classROOT_1_1Experimental_1_1XRooFit_1_1xRooNode.html) class, which provides a high-level API for interacting with RooFit/HS3 workspaces."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b5b12609-f8d8-4ff7-aebe-95b81fec7d9b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1mxRooFit -- Create/Explore/Modify Workspaces -- Development ongoing\u001b[0m \n",
+      "                xRooFit : http://gitlab.cern.ch/will/xroofit\n",
+      "                Version: 77eb847 [2024-03-19 10:27:42 +0100]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'6.31/01'"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ROOT\n",
+    "ROOT.gSystem.Load(\"xroofit_build/libxRooFit\")\n",
+    "from ROOT.Experimental import XRooFit\n",
+    "ROOT.gROOT.GetVersion() # show version of ROOT"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e3ac4de-9ee9-4d48-9d86-e92ed18e3806",
+   "metadata": {},
+   "source": [
+    "## 1.1. Opening a workspace and listing its content\n",
+    "\n",
+    "Start by creating an `xRooNode` from the file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ddebfcc8-55fe-47bd-9a4b-3b5afa67c167",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Warning in <xRooNode::xRooNode>: Inferring initial errors of 116 parameters (give all nuisance parameters an error to avoid this msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "w = ROOT.xRooNode(\"Unfolding_dPhi_ttbar_Z_5bins_comb_particle_tau2.4_NORM.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64a98d0e-a16f-40c4-8e92-5b5bbae081ed",
+   "metadata": {},
+   "source": [
+    "You can explore the content of the workspace somewhat like you would a file system: each node contains sub-nodes, which you can interact with to explore ever deeper. Here are some methods we"
```

Comment:
"Some methods we"?

Reply:
Given the complexity of the output (a new graduate student would have no idea what floating parameters vs parameters of interest are, what makes a parameter global or regular...), we should probably link to some documentation of the terminology? Maybe Wouter's paper?

Reply:
Will this suffice, do you think: https://xroofit.readthedocs.io/en/latest/basics.html (it's a work in progress, but I think covers what you want at this point)

Reply:
I think that's fantastic! And probably better than a paper if it's living.